controllers/business.js

```diff
@@ -0,0 +1,117 @@
+const { response } = require("express");
+const { Business } = require("../models/business");
+const {
+	addEntity,
+	getEntity,
+	updateEntity,
+	getWithFilter,
+} = require("../database/config");
+
+const healthy = (req, res) => {
+	res.status(200).json({
+		msg: `Healthy`,
+	});
+};
+
+const registerBusiness = async (req, res) => {
+	try {
+		const {
+      name,
+      username,
+      password,
+      businessType,
+      phone
+    } = req.body;
+
+    const business = new Business(
+      username,
+      name,
+      password,
+      businessType,
+      phone
+		);
+    
+    if (!await business.checkSanity()) {
+      throw new Error("Business is not valid");
+    }
+		await addEntity("Business", business.username, business);
+		
+    res.status(201).json({
+			msg: `Business ${business.name} added successfully`,
+		});
+	} catch (err) {
+		res.status(401).json({
+			msg: `Error adding business ${err}`,
+		});
+	}
+};
+
+const getBusiness = async (req, res = response) => {
```

Comment:
I thinks we should use the JWT with the validateJWT function to return the buissnes insted of using the params, would make it easier in the frontend